src/triggers.qc

```diff
@@ -488,10 +488,16 @@ teleport_touch =
 
 	other.angles = t.mangle;
 
+	if (other.classname == "shambler_slave") // Cobalt , fix Shambler from re-entering back to back telporters
+	{
+		other.teleport_time = time + 2.5;
+		other.velocity = v_forward * 600;
```

Comment:
This will prevent the repetitive respawns, but ultimately the issue is a setsize() one.